po/pt_BR.po

```diff
@@ -0,0 +1,153 @@
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:3
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:3 src/aboutDialog.ui:14
+msgid "Flatseal"
+msgstr "Flatseal"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:9
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:8
+msgid "A permissions manager for Flatpak"
+msgstr "Um gerenciador de permissões para aplicativos Flatpak"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:10
+msgid "com.github.tchx84.Flatseal"
+msgstr "com.github.tchx84.Flatseal"
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:10 src/aboutDialog.ui:12
+msgid ""
+"Flatseal is a graphical utility to review and modify basic permissions from "
+"your Flatpak applications."
+msgstr ""
+"O Flatseal é um utilitário gráfico para revisar e modificar permissões básicas de "
+"seus aplicativos Flatpak."
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:29
+msgid "Martin Abente Lahaye"
+msgstr "Martin Abente Lahaye"
+
+#: src/aboutDialog.ui:11
+msgid "Copyright © 2020 - Martín Abente Lahaye"
+msgstr "Copyright © 2020 - Martín Abente Lahaye"
+
+#: src/applicationRow.ui:57
+msgid "Application"
+msgstr "Aplicação"
+
+#: src/applicationRow.ui:74 src/permissionEntryRow.ui:45
+#: src/permissionSwitchRow.ui:45
+msgid "Description"
+msgstr "Descrição"
+
+#: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
+msgid "Pemission"
+msgstr "Permissão"
+
+#: src/view.ui:36
+msgid "No applications found."
+msgstr "Nenhum aplicativo encontrado."
+
+#: src/window.ui:41
+msgid "Reset"
+msgstr "Redefinir"
+
+#: src/window.ui:47
+msgid "Reset this application permisions"
+msgstr "Redefinir as permissões deste aplicativo"
+
+#: src/model.js:35
+msgid "Access network"
+msgstr "Acesso a rede"
+
+#: src/model.js:40
+msgid "Access inter-process communications"
+msgstr "Acesse comunicações entre processos"
+
+#: src/model.js:45
+msgid "Access X11 windowing system"
+msgstr "Acesso ao sistema de janelas X11"
+
+#: src/model.js:50
+msgid "Access X11 windowing system (as fallback)"
+msgstr "Acesso ao sistema de janelas X11 (como substituto)"
+
+#: src/model.js:55
+msgid "Access Wayland windowing system"
+msgstr "Acesso ao sistema de janelas Wayland"
+
+#: src/model.js:60
+msgid "Access PulseAudio sound server"
+msgstr "Acesso ao servidor de som PulseAudio"
+
+#: src/model.js:65
+msgid "Access D-Bus system bus (unrestricted)"
+msgstr "Acesso ao barramento do sistema D-Bus (irrestrito)"
+
+#: src/model.js:70
+msgid "Access D-Bus session bus (unrestricted)"
+msgstr "Acesso ao barramento de sessão do D-Bus (irrestrito)"
+
+#: src/model.js:75
+msgid "Access Secure Shell agent"
+msgstr "Acessar agente Secure Shell"
+
+#: src/model.js:80
+msgid "Access printing system"
+msgstr "Acesso ao sistema de impressão"
+
+#: src/model.js:85
+msgid "Access GPU acceleration"
+msgstr "Aceleração de acesso à GPU"
```

Comment:
I need to practice my portuguese, can you can explain why isn't "Acesso" to "aceleração"? Like the other translations.